packages/openapi3/src/schema-emitter-3-0.ts

```diff
@@ -206,12 +208,28 @@ export class OpenAPI3SchemaEmitter extends OpenAPI3SchemaEmitterBase<OpenAPI3Sch
           (schema instanceof Placeholder || "$ref" in schema) &&
           !(type && shouldInline(program, type))
         ) {
-          if (type && (type.kind === "Model" || type.kind === "Scalar")) {
+          if (type && type.kind === "Model") {
             return new ObjectBuilder({
               type: "object",
               allOf: Builders.array([schema]),
               ...additionalProps,
             });
+          } else if (type && type.kind === "Scalar") {
+            const stdType = $.scalar.getStdBase(type);
+            let outputType: JsonType | undefined = undefined;
+            if (stdType !== null) {
+              const scalarEx = {
```

Comment:
Sa why do you need to do that ?

Reply:
Since `getSchemaForStdScalars(scalar: Scalar & { name: IntrinsicScalarName }): Schema`, otherwise I need to convert to any `as any`?

Reply:
Yeah just as any, if you cheating the system already don't need to create another spread. Can also get rid of the if and use ?.

Reply:
updated